src/app/_components/Header.js

```diff
@@ -38,7 +38,9 @@ export function Header() {
     setDonateModalOpen(prevState => !prevState);
   }, [setDonateModalOpen]);
 
-  useBodyScrollLock(isMenuOpen, 'y');
+  const isScrollLock = isMenuOpen || isFeedbackOpen || isDonateModalOpen;
+  useBodyScrollLock(isScrollLock);
```

Comment:
I don't really like it, I prefer when this logic is inside a modal/drawer component, so no matter where do we render them they'll behave the same

Reply:
Finally figured out how to do it, thanks! Now it should look better